ui/src/services/job/JobStatus.js

```diff
@@ -3,22 +3,22 @@ import { Enum, EnumValue } from "../enum/Enum";
 export const JobStatus = Enum({
   PENDING: EnumValue("pending", {
     label: "Pending",
-    color: "#FEC514",
+    color: "warning",
```

Comment:
Changing these colours to use the default EUI colour template wherever possible.